doc/DEVELOPMENT.md

```diff
@@ -0,0 +1,44 @@
+# Radar
+
+This document guides into different development tasks without installing and running the complete production system on a personal workstation.
+
+The core of the system is written in [Python 3](https://www.python.org/) and installing that is the minimum requirement. Then, clone the project and get started!
+
+## Set up python virtual environment
+
+Following terminal commands create an environment and install required packages.
+
+Linux & OS X:<br>
+`python3 -m virtualenv -p python3 py_venv`<br>
+`source py_venv/bin/activate`<br>
+`pip install -r requirements.txt`<br>
+Windows:<br>
+`C:\Python35\python -m venv py_venv`<br>
+`py_venv\Scripts\activate.bat`<br>
+`pip install -r requirements.txt`
+
+When starting a new terminal for development, access the same virtual enviroment by enterint the activate-command alone (2nd line).
```

Comment:
typo enterint -> entering

Reply:
Should be fixed by @Hucisko who controls the branch now. Also my commits can be squashed if desired.

Reply:
Additionally, I think there is no hyphen in `the activate command`. :smile: